content/en/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -65,7 +65,7 @@ in some cases, the same namespace. For example, WordPress can be installed more
 than once where different websites are different installations of WordPress.
 
 The name of an application and the instance name are recorded separately. For
-example, WordPress has a `app.kubernetes.io/name` of `wordpress` while it has
+example, WordPress has an `app.kubernetes.io/name` of `wordpress` while it has
```

Comment:
Hello @xfc .
You could accept the suggested changes, or clarify the text a bit more.